src/components/augmentedReality/AugmentedReality.js

```diff
@@ -38,22 +37,22 @@ export const AugmentedReality = ({ navigation, onSettingsScreen, tourID }) => {
   }, []);
 
   useEffect(() => {
-    setData(staticData);
+    setData(tourStops);
 
-    if (staticData?.length) {
-      checkDownloadData({ data: staticData });
+    if (tourStops?.length) {
+      checkDownloadData({ data: tourStops });
     }
-  }, [staticData]);
+  }, [tourStops]);
 
   const checkDownloadData = async ({ data }) => {
     setIsLoading(true);
     await checkDownloadedData({ data, setData });
     setIsLoading(false);
   };
 
-  if (error || !isARSupported) return null;
+  if (!isARSupported) return null;
 
-  if (isLoading || !staticData) return <LoadingSpinner loading />;
+  if (isLoading || !tourStops?.length) return <LoadingSpinner loading />;
```

Comment:
when i look at this now, is it always correct to show a loading spinner if there are no tour stops?

Reply:
In fact, this component is not displayed when `tourStops is not present. This place can be organised.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/afa7f49f3c7659b48f0bc217d6e549966e443c24